bench/bench-perf.hs

```diff
@@ -0,0 +1,410 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+module Main where
+
+import Control.Monad.State.Strict (liftIO)
+import Data.Aeson qualified as JSON
+import Data.ByteString.UTF8 as BSU 
+import Data.ByteString qualified as ByteString
+import Data.Either
+import Data.Function
+import Data.Map.Strict qualified as Map
+import Data.Maybe
+import Data.String.Here
+import EVM
+import EVM.ABI
+import EVM.Effects (App, runApp)
+import EVM.Fetch qualified as Fetch
+import EVM.Format
+import EVM.Sign
+import EVM.Solidity
+import EVM.Stepper qualified as Stepper
+import EVM.Test.Tracing
+import EVM.Transaction qualified
+import EVM.Types
+import EVM.UnitTest
+import GHC.Word
+import System.Directory (setCurrentDirectory)
+import System.FilePath.Find qualified as Find
+import System.IO.Temp
+import System.Process (readProcessWithExitCode)
+import Test.Tasty.Bench
+import Control.Monad.ST
+import EVM.FeeSchedule (feeSchedule)
+
+-- benchmark hevm and EVM tool on the folder called benchmarks
```

Comment:
I... don't understand. Where is this folder? It's not part of the PR I think? Am I supposed to have it? Sorry I must be missing something about tasty-bench :S

Reply:
I saw this implements some functions to run things like geth and evm but I'm not sure how they're supposed to be used. Maybe it's referring to that, or maybe it's dead code 🤔

Reply:
I suspect some dead code related to what's below. Let's remove it all -- they don't seem to serve a purpose for the moment I think?